src/main/java/com/eerussianguy/blazemap/api/maps/MapType.java

```diff
@@ -20,7 +20,7 @@
  *
  * @author LordFokas
  */
-public abstract class MapType implements BlazeRegistry.RegistryEntry {
+public abstract class MapType implements BlazeRegistry.RegistryEntry, IMapPart {
```

Comment:
add `@Override` to the interface methods?

Reply:
- [ ] Add `@Override` to interface methods on `MapType`